egs/iam/s5/local/score.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
```

Comment:
this should be a symlink to steps/score_kaldi.sh, not a copy

Reply:
It's because it calls `local/unk_arc_post_to_transcription.py` somewhere
in the scoring pipeline. However, looking at the script, 
it seems to me that it can be done
through `hyp_filtering_cmd`. @aarora8, could you please make it
a symlink and do the "unk scoring" using `hyp_filtering_cmd`?

Reply:
ok, it is also converting upper case to lower case. But in the mail response Paul from RWTH-aachen mention that   they are not converting upper case to lower case. should I not include the conversion in the change as well.